dtfs-central-api/src/cron-scheduler-jobs/create-utilisation-reports/index.test.ts

```diff
@@ -13,7 +13,7 @@ jest.mock('@ukef/dtfs2-common', () => ({
   getCurrentReportPeriodForBankSchedule: jest.fn(),
 }));
 
-const originalProcessEnv = process.env;
+const originalProcessEnv = { ...process.env };
 
 describe('scheduler/jobs/create-utilisation-reports', () => {
   afterEach(() => {
```

Comment:
Is it worth keeping the `afterEach`'s consistent across files with  `process.env = originalEnv` as opposed to the second spread?  (You've done it some places, I think you just missed it in this file and a few others! :))